File: chatwoot/python/src/UpdateAgentInAccountExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
)

with ApiClient(configuration) as api_client:
    update_agent_in_account_request = models.UpdateAgentInAccountRequest(
        role=None,
        availability=None,
        auto_offline=None,
    )

    try:
        response = api.AgentsApi(api_client).update_agent_in_account(
            account_id=None,
            id=None,
            update_agent_in_account_request=update_agent_in_account_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling AgentsApi#update_agent_in_account: %s\n" % e)

File: chatwoot/ruby/src/UpdateAgentInAccountExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
end

update_agent_in_account_request = ChatwootClient::UpdateAgentInAccountRequest.new
update_agent_in_account_request.role = "agent"

begin
    response = ChatwootClient::AgentsApi.new.update_agent_in_account(
        0, # account_id
        0, # id
        update_agent_in_account_request, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling AgentsApi#update_agent_in_account: #{e}"
end
